sources/api/early-boot-config/build.rs

```diff
@@ -4,8 +4,31 @@ use std::env;
 use std::fs::File;
 use std::io::Write;
 use std::path::PathBuf;
+use std::process;
 
 fn main() {
+    // The code below emits `cfg` operators to conditionally compile this program based on the
+    // current variant.
+    // TODO: Replace this approach when the build system supports ideas like "variant
+    // families": https://github.com/bottlerocket-os/bottlerocket/issues/1260
```

Comment:
```suggestion
    // TODO: Replace this approach when the build system supports ideas like "variant
    // tags": https://github.com/bottlerocket-os/bottlerocket/issues/1260
```

I think we are preferring a more general, or flatter, conception currently.